admin_manual/appliance/Office.rst

```diff
@@ -13,6 +13,9 @@ Here is an overview of the process:
 
 .. warning::  Access with **HTTPS** using **domain name** is required.
 
+.. tip::
+If you encounter the error, "Failed to load the document. Please endure the file type is supported and not corrupted, and try again.", when trying to open documents, either restart the Collabora Docker container or the appliance.
```

Comment:
s/Please endure/Please ensure/g

Reply:
:D  maybe I meant that they have to endure it :)